lib/chef-acceptance/suite_runner.rb

```diff
@@ -0,0 +1,94 @@
+require 'chef-acceptance/output_formatter'
+require 'chef-acceptance/chef_runner'
+
+module ChefAcceptance
+  # Responsible for running one or more suites and displaying statistics about
+  # the runs to the user
+  class SuiteRunner
+    attr_reader :force_destroy
+
+    def initialize(force_destroy = false)
+      @force_destroy = force_destroy
+    end
+
+    def run(suite_regex, command)
+      output_formatter = OutputFormatter.new
+      suite_stats = SuiteStatistics.new
+      suites = find_suites(suite_regex)
```

Comment:
We should not worry about the `suite_regex` for now. Depending on our priorities we might want to pick a different item as the next thing to do which will make us live with this extra code for a while.